server/src/main/java/io/crate/planner/operators/InsertFromValues.java

```diff
@@ -406,32 +406,24 @@ public CompletableFuture<BulkResponse> executeBulk(DependencyCarrier dependencie
                     plannerContext,
                     dependencies.clusterService());
 
-            try {
-                Iterator<Row> rows = evaluateValueTableFunction(
-                    tableFunctionRelation.functionImplementation(),
-                    tableFunctionRelation.function().arguments(),
-                    writerProjection.allTargetColumns(),
-                    tableInfo,
-                    param,
-                    plannerContext,
-                    subQueryResults);
+            Iterator<Row> rows = evaluateValueTableFunction(
+                tableFunctionRelation.functionImplementation(),
+                tableFunctionRelation.function().arguments(),
+                writerProjection.allTargetColumns(),
+                tableInfo,
+                param,
+                plannerContext,
+                subQueryResults);
 
-                while (rows.hasNext()) {
-                    Row row = rows.next();
-                    Item item = grouper.apply(shardedRequests, row, true);
+            while (rows.hasNext()) {
+                Row row = rows.next();
+                Item item = grouper.apply(shardedRequests, row, true);
 
-                    // Primary Key and CLUSTERED BY check is already done in grouper -> RowShardResolver, both cannot be null.
-                    // constraintsChecker is also used in grouper.apply but only for partitioned tables.
-                    // We keep the check below so that insert from values into regular tables doesn't send invalid values to the shards and fails early.
-                    constraintsChecker.accept(indexNameResolver.get(), item);
-                    bulkIndices.add(bulkIdx);
-                }
-            } catch (Throwable t) {
-                for (int i = 0; i < bulkResponse.size(); i++) {
-                    bulkResponse.setFailure(i, t);
-                }
-                result.complete(bulkResponse);
```

Comment:
stuck jobs test is failing - should we do smth like `result.completeExceptionally(t)` so that Session takes care of cleaning up sys.jobs?